package.json

```diff
@@ -20,15 +20,6 @@
         "copy-scripts:array": "cp node_modules/posthog-js/dist/array.js* frontend/dist/",
         "copy-scripts:recorder": "cp node_modules/rrweb/dist/record/rrweb-record.min.js frontend/dist/recorder.js && cp node_modules/rrweb/dist/record/rrweb-record.min.js.map frontend/dist/recorder.js.map && cat node_modules/rrweb/dist/plugins/console-record.min.js >> frontend/dist/recorder.js && sed -i -e s/rrweb-record.min.js.map/recorder.js.map/ frontend/dist/recorder.js && sed -i -e 's/\\/\\/\\# sourceMappingURL=recorder/window.rrweb = {record: rrwebRecord}\\n\\/\\/# sourceMappingURL=recorder/g' frontend/dist/recorder.js",
         "test": "jest",
-        "start": "concurrently -n ESBUILD,TYPEGEN -c yellow,green \"yarn start-http\" \"yarn run typegen:watch\"",
-        "start-http": "yarn start-http:esbuild",
-        "start-http:esbuild": "yarn clean && yarn copy-scripts && node frontend/build.mjs --dev",
-        "start-http:webpack": "mkdir -p frontend/dist/ && cp -a frontend/public/* frontend/dist/ && npm run copy-scripts && webpack serve",
-        "start-https": "yarn start-https:webpack",
-        "start-https:webpack": "yarn start-http:webpack --https",
-        "start-docker": "yarn start-docker:esbuild",
-        "start-docker:esbuild": "yarn start-http:esbuild --host 0.0.0.0",
-        "start-docker:webpack": "yarn start-http:webpack --host 0.0.0.0",
         "start-ch-dev": "concurrently -n DOCKER,ESBUILD,TYPEGEN -c red,blue,green \"docker-compose -f docker-compose.dev.yml pull && CH_WEB_SCRIPT=./ee/bin/docker-ch-dev-backend docker-compose -f docker-compose.dev.yml up\" \"yarn run start-http --host 0.0.0.0\" \"yarn run typegen:watch\"",
```

Comment:
Now that's a bit too much... We only want to yeet webpack. Fixing in a followup.

Reply:
I thought that too but if I'm not wrong I've removed exactly the lines you've highlighted in your previous comment

Reply:
ah... the classic "Go buy bread. If they have eggs, buy six" --> comes home with 6 loafs of bread 🙈 

`yarn start` is still something we want